lib/systems/default.nix

```diff
@@ -34,6 +84,8 @@ rec {
   */
   flakeExposed = import ./flake-systems.nix { };
 
+  inherit canExecute emulatorAvailable emulator;
```

Comment:
Why not just have the definitions here, instead of defining them above and then inheriting them into a recursive attrset?

Reply:
3c760ba7ab9a